src/components/molecules/LogOutput/LogOutput.styled.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components';
+
+import Colors from '@styles/Colors';
+
+const invisibleScroll = `
+  &::-webkit-scrollbar {
```

Comment:
Maybe remove this as you defined a global style for it and import it from there?